.github/workflows/CI-python.yml

```diff
@@ -13,20 +13,20 @@ jobs:
     strategy:
```

Comment:
can you add the version bump as well to the PR?

Reply:
Version bump for rai_core_flask?

Reply:
Spoke offline to Ilya. We need not update the version for this PR as there are no namepsace changes.